pygame/_error.py

```diff
@@ -24,8 +24,13 @@ def unpack_rect(rect):
 
 
 def get_error():
-    return ffi.string(sdl.SDL_GetError())
+    err = ffi.string(sdl.SDL_GetError())
+    if not isinstance(err, str):
+        return err.decode('utf8')
```

Comment:
This returns a string in python 3, and a string / bytes object in python 2, which matches the behaviour of pygame on python 3.

Whether it's a good idea or not is less clear. but since we're aiming to match pygame, it seems best to do this as well.